src/forklift/core.py

```diff
@@ -225,7 +235,7 @@ def _hash(crate, hash_path, needs_reproject):
     if not arcpy.Exists(path.join(hash_path, crate.name)):
         log.debug('%s does not exist. creating', crate.name)
         table = arcpy.CreateTable_management(hash_path, crate.name)
-        arcpy.AddField_management(table, hash_id_field, 'TEXT', field_length=50)
+        arcpy.AddField_management(table, hash_id_field, 'LONG')
```

Comment:
Yes. The hash id field is a foreign key for the destination. So it doesn't depend on the source and destination should always be in a geodatabase.